### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# @microsoft/signalr package API Reference Docs
+
+This repo contains documentation for the obsolete [@microsoft/signalr package](https://docs.microsoft.com/en-us/javascript/api/@microsoft/signalr) API reference. The current repo is https://github.com/MicrosoftDocs/signalr-script-pr and this repo is no longer used.
```
Comment (BrennanConroy) [inline README.md:3]: @aspnet/signalr

Comment (BrennanConroy) [reply README.md:3]: @Rick-Anderson

Comment (Rick-Anderson) [inline README.md:3]:
```suggestion
This repo contains documentation for the obsolete [@aspnet/signalr package](https://docs.microsoft.com/en-us/javascript/api/@microsoft/signalr) API reference. The current repo is https://github.com/MicrosoftDocs/signalr-script-pr and this repo is no longer used.

```

Comment (Rick-Anderson) [reply README.md:3]: @BrennanConroy  commit suggestion or provide an alternative suggestion..

Comment (BrennanConroy) [reply README.md:3]:
```suggestion
This repo contains documentation for the obsolete [@aspnet/signalr package](https://learn.microsoft.com/javascript/api/@aspnet/signalr) API reference. The current repo is https://github.com/MicrosoftDocs/signalr-script-pr and this repo is no longer used.

```

**Review by BrennanConroy — APPROVED** [review]

**Pull request merged**